llvm/lib/Target/X86/X86InstrSSE.td

```diff
@@ -6699,28 +6699,30 @@ let Constraints = "$src1 = $dst" in {
                                    null_frag>, REX_W;
   }
 
+  let Predicates = [In64BitMode] in {
```

Comment:
This `let` is incorrect. You will override the default predicate `[HasCRC32]`

Reply:
Nice catch, will fix.